cmd2.py

```diff
@@ -2241,13 +2244,11 @@ def do_shell(self, command):
         proc.communicate()
 
     @staticmethod
-    def _get_exes_in_path(starts_with, at_eol):
+    def _get_exes_in_path(starts_with):
```

Comment:
But it looks like you are making some changes which are intended to help with that PR.  So I guess we can merge this in and then merge master into that PR to pick up the unit tests in that PR (once they are fixed).